pkg/archive/archive.go

```diff
@@ -217,6 +230,13 @@ func includeFile(fpath string) bool {
 	return split[0] == config.InternalDir || split[0] == config.PublishDir || split[0] == "catalogs"
 }
 
+func delete(fpath string, info fs.FileInfo) bool {
```

Comment:
Can you call this something else so there's no conflict with the builtin `delete()` function?
```suggestion
func shouldRemove(fpath string, info fs.FileInfo) bool {
```

Reply:
Good point. I will change the name of this.